src/App.css

```diff
@@ -1,106 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;700&family=Roboto:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,700;1,100;1,200;1,300&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
   list-style: none;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+a {
+  font-family: Roboto Slab, sans-serif;
+  color: #4386bf;
 }
 
 body {
-  background-color: #f1f1f1;
+  background-color: #fafafa;
+  font-family: 'Roboto', sans-serif;
 }
 
-.newbook {
+nav > button {
+  margin-left: auto;
+}
+
+form {
   display: flex;
-  margin: auto;
-  flex-direction: column;
-  justify-content: center;
   align-items: center;
   gap: 1em;
 }
 
+.newbook {
+  border-top: solid 1px #e8e8e8;
+  margin-left: 1%;
+  padding-bottom: 14px;
+  width: 100%;
+  margin-top: 3%;
+}
+
+.newbook > h3 {
+  font-size: 1.25rem;
+  font-family: 'Montserrat', sans-serif;
+  height: 1rem;
+  width: 50%;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
 .newbook button {
-  margin-top: 10px;
   font-size: 12px;
-  background-color: rgb(25, 173, 192);
+  background-color: rgb(2, 144, 255);
   color: white;
   border: none;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  width: 18%;
+  height: 2.713rem;
+  font-family: Roboto Slab, sans-serif;
+  font-weight: bold;
+  letter-spacing: 0.1em;
+}
+
```

Comment:
![image](https://github.com/Katended/book-store/assets/93039359/b5e690ba-6d65-4932-a7d9-2c3060f42eee)

- [ ] It would be nice if you make your design look similar to the given design because while fixing the requested changes you have made your Current Chapter heading to much close to the upper edge which is not according to the given design 😨 Kindly make your design look similar to the given design to make your project more professional 👍
- [ ] It would be nice If you make your button width as in the given design to make your project look attractive and follow the project requirements 😊